PKHeX.WinForms/Subforms/PKM Editors/MemoryAmie.cs

```diff
@@ -435,10 +435,10 @@ private void changeCountryText(object sender, EventArgs e)
 
         private void update255_MTB(object sender, EventArgs e)
         {
-            MaskedTextBox mtb = sender as MaskedTextBox;
+            MaskedTextBox mtb = (MaskedTextBox) sender;
```

Comment:
Casting using `as` is faster than type conversion.
https://www.codeproject.com/articles/8052/type-casting-impact-over-execution-performance-in